locales/en.json

```diff
@@ -126,21 +126,42 @@
   "Request types fetched successfully": "Request types fetched successfully",
   "Location id must be a number": "Location id must be a number",
   "Can not find your line manager": "Can not find your line manager",
-  "Name should be string": "Name should be string",
-  "Name input can't be empty": "Name input can't be empty",
-  "Name property is required": "Name property is required",
-  "Location should be string": "Location should be string",
-  "Location input can't be empty": "Location input can't be empty",
-  "Location should be provided": "Location should be provided",
-  "Description should be of type string": "Description should be of type string",
-  "Please, provide meaningful description": "Please, provide meaningful description",
-  "At least one room is required": "At least one room is required",
-  "Please provide a valid url": "Please provide a valid url",
-  "Provide image for your accomodation": "Provide image for your accomodation",
-  "Image is required": "Image is required",
-  "Accomodation created successfully": "Accomodation created successfully",
-  "Room in facility name created successfuly": "Room in facility name created successfuly",
-  "Accomodation successfully updated": "Accomodation successfully updated",
-  "Room Deleted successfully": "Room Deleted successfully",
-  "Accomodation deleted successfully": "Accomodation deleted successfully"
+	"Location is required": "Location is required",
+	"Destination is required": "Destination is required",
+	"Start date is required": "Start date is required",
+	"End date is required": "End date is required",
+	"Travel reasons is required": "Travel reasons is required",
+	"Approval email is required": "Approval email is required",
+	"Approval status  is required": "Approval status  is required",
+	"Trip requested success": "Trip requested success",
+	"Successfully deleted trip": "Successfully deleted trip",
+	"Success updated trip": "Success updated trip",
+	"Accomodation is required": "Accomodation is required",
+	"Type is required": "Type is required",
```

Comment:
Look out for these spacing changes!

Reply:
It is prettier it's self but i am going to work on it

Reply:
solved